cli.bats

```diff
@@ -37,14 +37,14 @@ teardown() {
     [[ "$output" == *"SUBCOMMANDS"* ]]
 }
 
-@test "succeeds: --chain=morden account new [empty options]" {
-    run $EMERALD_CLI --chain=morden account new <<< $'foo\n'
+@test "succeeds: --chain=etc-test account new [empty options]" {
+    run $EMERALD_CLI --chain=etc-test account new <<< $'foo\n'
     [ "$status" -eq 0 ]
     [[ "$output" == *"Created new account"* ]]
 }
 
-@test "succeeds: --chain=mainnet new --security=high --name='Test account' --description='Some description'" {
-    run $EMERALD_CLI --chain=mainnet \
+@test "succeeds: --chain=etc-main new --security=high --name='Test account' --description='Some description'" {
+    run $EMERALD_CLI --chain=etc-main \
```

Comment:
why?

Reply:
becaus eth and etc can't share same name

Reply:
see above